src/tf-types.ts

```diff
@@ -0,0 +1,173 @@
+import { SupportTable } from './factories/factory-types'
+import {
+  BlankNodeTermType,
+  DefaultGraphTermType,
+  LiteralTermType,
+  NamedNodeTermType,
+  VariableTermType,
+} from './types'
+
+/**
+ * RDF/JS taskforce Term
+ * @link https://rdf.js.org/data-model-spec/#term-interface
+ */
+export interface TFTerm {
```

Comment:
Let's get rid of the entire `TF` prefix; it's not the right one, and it doesn't have a place here. rdflib.js terms are RDF/JS-compatible.

Reply:
If we remove the TF prefix, how should we distinguish between RDFlib.js NamedNode (which has some extra methods) and RDF/JS NamedNodes? Add a prefix for RDFLib types?

Reply:
A prefix for RDFLib types that extend RDF/JS types would make sense to me, since they're the special case in this regard.

Reply:
RDFLib types, however, are actually Classes. Do you suggest I rename the classes? `NamedNode` => `RDFlibNamedNode`, or perhaps rename the imports?

```js
import {NamedNode as RDFlibNamedNode} from "./namednode"
```

Don't think this is what you'd want.

Reply:
We need to distinguish between the export names of the library (as a whole) and the internal naming. The external naming won't change in any case; the exported `NamedNode` will be the rdflib one. I think for internal purposes, `RDFlibNamedNode` makes sense. Or perhaps `RdfLib.NamedNode` or so.

Reply:
Changed the names!